R/pkg/NAMESPACE

```diff
@@ -287,6 +287,7 @@ export("as.DataFrame",
        "read.json",
        "read.parquet",
        "read.text",
+       "sparkLapply",
```

Comment:
that would conflict with base::lapply - in other words it would prevent the user from calling lapply on R native data even when it has nothing to do with Spark, in the session.
(longer explanation: since this is S3, the method routing is by name, so having the same name in a package loaded later (SparkR) would override the one in the base package which is loaded earlier)

I'd like lapply better though ;)